infrastructure/variables.tf

```diff
@@ -77,3 +77,7 @@ variable "ftp_reports_folder" {
 variable "ftp_reports_cron" {
   default = "0 30 0 * * *"
 }
+
+variable "deployment_namespace" {
+  default = ""
+}
```

Comment:
should it not be `variable "deployment_namespace" { }` instead?

Reply:
I have followed what was done in rhubarb-service so not sure if we don't have default it would work.Can give it a try though.